Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 #Build application stage
 #We need maven.
-FROM maven:3.8.4-jdk-11
+FROM maven:3.8.4-eclipse-temurin-17
```

Comment:
Is there a particular reason to use this docker image ?

Reply:
I was more thinking about "why eclipse temurin", is there any particular reason to choose this release ? maybe it's the one that is commonly used with spring boot 3 by default ?

Reply:
Eclipse temurin is an OSS organization / project providing up to date standard OpenJDK builds, which are not provided by oracle anymore. It is the new name of the "AdoptJDK" builds